.github/workflows/validate.yml

```diff
@@ -10,7 +10,8 @@ jobs:
       with:
         ruby-version: '3.2' # Not needed with a .ruby-version file
         bundler-cache: true # runs 'bundle install' and caches installed gems automatically
-    - run: gem install cbor-diag cddl
+    - run: gem install cbor-diag
+    - run: gem install cddl -v 0.10.3
```

Comment:
Some how the latest cddl tool 0.10.5 fails with an error, so we use 0.10.3 instead.